server/sonar-server/src/main/java/org/sonar/server/permission/PermissionTemplateService.java

```diff
@@ -74,8 +69,13 @@ public void applyDefaultPermissionTemplate(String componentKey) {
     }
   }
 
+  /**
+   * @deprecated replaced by {@link #applyDefault(DbSession, ComponentDto, Long)}, which <b>does not
+   * verify that user is authorized to administrate the component</b>.
+   */
+  @Deprecated
   public void applyDefaultPermissionTemplate(DbSession session, String componentKey) {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)